subprojects/docs/src/docs/userguide/gradle_wrapper.adoc

```diff
@@ -238,3 +238,41 @@ distributionSha256Sum=371cb9fbebbe9880d147f59bab36d61eee122854ef8c9ee1ecf12b8236
 ----
 
 Gradle will report a build failure in case the configured checksum does not match the checksum found on the server for hosting the distribution. Checksum Verification is only performed if the configured Wrapper distribution hasn't been downloaded yet.
+
+[[wrapper_checksum_verification]]
+== Verifying the checksum of the Gradle Wrapper JAR
+
+Since the Wrapper JAR file is usually checked in to a project's version control system, verifying its checksum provides a simple way to check its integrity.
+In addition, as with all binary artifacts, it's a way to ensure that it has not been tampered with, for example when reviewing a pull request that upgrades the Wrapper JAR in an open source project.
+The checksum of the Wrapper JAR is published alongside the corresponding Gradle distribution on https://services.gradle.org/.
```

Comment:
> The checksums _for all versions_ of the Wrapper JAR [since whatever version] are published alongside the ...

Make it clear that readers can find more than one checksum at that URL.